litex/tools/litex_json2dts.py

```diff
@@ -12,7 +12,7 @@
 import argparse
 
 
-def generate_dts(d):
+def generate_dts(d, initrd_start_offset, initrd_size):
```

Comment:
Maybe set `None` as default values for both `initrd_start_offset` and `initrd_size` to make this function backwards-compatible?

Reply:
Sure.

Reply:
Fix this.